yarn-project/end-to-end/src/deploy_l1_contracts.ts

```diff
@@ -65,6 +73,14 @@ export const deployL1Contracts = async (
   };
 };
 
+/**
+ * Helper function to deploy ETH contracts.
+ * @param walletClient - A viem WalletClient.
+ * @param publicClient - A viem Public client.
```

Comment:
```suggestion
 * @param publicClient - A viem PublicClient.
```

This was triggering my OCD 😆